src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -895,11 +895,11 @@ class JsonLinePrinter : public JsonPrinter {
         BSLS_KEYWORD_OVERRIDE
     {
         closeBraceIfOpen();
-        d_ostream << "  \"TotalRecordsNumber\": \"" << totalRecordsCount
-                  << "\",\n";
+        d_ostream << "  \"totalRecordsNumber\": " << totalRecordsCount
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.